build.gradle

```diff
@@ -19,6 +19,7 @@ import org.gradle.build.BuildTypes
 
 defaultTasks 'assemble'
 apply plugin: 'java-base'
+apply plugin: 'sonar-runner'
```

Comment:
My goal was to be able to analyze Gradle itself with sonarRunner task as a proof of concept (eat your own dogfood). But probably I should have proposed this change in a separate pull request. I'll remove this change.